docs/howto/writing-migrations.txt

```diff
@@ -271,3 +271,51 @@ Prefer using ``dependencies`` over ``run_before`` when possible. You should
 only use ``run_before`` if it is undesirable or impractical to specify
 ``dependencies`` in the migration which you want to run after the one you are
 writing.
+
+
+Migrating data when replacing an external app
+=============================================
+
+If you plan to move from one external application to another one with a similar
+data structure, you can use a data migration. If you plan to remove the old 
+application later, you will need to set the ``dependencies`` property
+dynamically. Otherwise you will have missing dependencies once you uninstall
+the old application::
```

Comment:
I'd recon to use the `.. snippet::` syntax instead to make file names a bit more clear:

```rst
the old application:

.. snippet::
    :filename: myapp/migrations/0002_ensure_dependencies.py

    from django.db import migrations
    # ...
```